packages/trader/src/AppV2/Containers/Trade/trade-types.tsx

```diff
@@ -251,57 +284,26 @@ const TradeTypes = ({ contract_type, onTradeTypeSelect, trade_types, is_dark_mod
-                                variant='secondary'
-                                className='draggable-list-category-header-button'
-                                onClick={is_editing ? savePinnedToLocalStorage : handleCustomizeTradeTypes}
-                            >
-                                {is_editing ? (
-                                    <Localize i18n_default_text='Done' />
-                                ) : (
-                                    <Localize i18n_default_text='Customise' />
-                                )}
-                            </Button>
-                        </div>
-                    </div>
-                    <ActionSheet.Content className='trade-types-dialog__content'>
-                        {is_editing ? (
-                            <DraggableList
-                                categories={pinned_trade_types}
-                                onRightIconClick={handleRemovePinnedClick}
-                                onAction={savePinnedToLocalStorage}
-                                onDrag={handleOnDrag}
-                            />
-                        ) : (
-                            <TradeTypeList
-                                categories={pinned_trade_types}
-                                onTradeTypeClick={handleOnTradeTypeSelect}
-                                isSelected={id => isTradeTypeSelected(id)}
-                                selectable
+                    {is_editing ? (
+                        <React.Fragment>
+                            <ActionSheet.Header
+                                title={
+                                    <div className='trade-types-dialog__title'>
+                                        <Localize i18n_default_text='Trade types' />
+                                    </div>
+                                }
                             />
-                        )}
-                        <TradeTypeList
-                            categories={other_trade_types}
-                            onRightIconClick={is_editing ? handleAddPinnedClick : undefined}
-                            onTradeTypeClick={!is_editing ? handleOnTradeTypeSelect : undefined}
-                            isSelected={id => isTradeTypeSelected(id)}
-                            selectable={!is_editing}
+                            <TradeTypesContent {...trade_type_content_props} />
+                        </React.Fragment>
+                    ) : (
+                        <Carousel
+                            header={CarouselHeader}
+                            pages={action_sheet_content}
+                            title={<Localize i18n_default_text='Trade types' />}
+                            next_icon={LabelPairedPresentationScreenSmRegularIcon}
+                            onNextButtonClick={() => sendOpenGuideToAnalytics(contract_type, 'trade_type_page')}
```

Comment:
Added function for analytics collection. Previously it was inside Guide button, but now we are not rendering this part.

Reply:
cc: @vinu-deriv